src/reconcile/rec_child.c

```diff
@@ -143,23 +143,24 @@ __rec_child_deleted(
      * cells to the page. Copy out the current fast-truncate information for that function.
      */
     if (!visible_all) {
-        /*
```

Comment:
@kommiharibabu I think it is this change. We should not revert this.

Reply:
We made a mistake in the WT-10307 pr. It should remove this section instead of only removing the if check. This accidentally reduces the disk space usage.